.evergreen/run-unit-tests.sh

```diff
@@ -3,18 +3,16 @@
 set -o xtrace   # Write all commands first to stderr
 set -o errexit  # Exit the script with error if any of the commands fail
 
-# Supported/used environment variables:
-#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java "jdk8"
-
-JDK=${JDK:-jdk}
-export JAVA_HOME="/opt/java/${JDK}"
+export JDK8="/opt/java/jdk8"
+export JDK17="/opt/java/jdk17"
+export JAVA_HOME=$JDK17
 
 ############################################
 #            Main Program                  #
 ############################################
 
-
-echo "Running tests with ${JDK} connecting to $MONGODB_URI"
+echo "Running unit tests on JDK17 and JDK8"
 
 ./gradlew -version
 ./gradlew --stacktrace --info test
+./gradlew --stacktrace --info testsOnJava8
```

Comment:
Should we also be running integration tests on Java 8?

Reply:
I'm not sure we need to, I just want to ensure we can run on Java 8

Reply:
I just wasn't sure how much unit test coverage there is